src/pybind/mgr/cephadm/module.py

```diff
@@ -2121,9 +2122,9 @@ def _get_container_image_info(self, image_name) -> ContainerInspectInfo:
             )
             self.log.debug(f'image {image_name} -> {r}')
             return r
-        except (ValueError, KeyError) as _:
-            msg = 'Failed to pull %s on %s: %s' % (image_name, host, '\n'.join(out))
-            self.log.exception(msg)
+        except (ValueError, KeyError) as e:
+            msg = 'Failed to pull %s on %s: Cannot decode JSON: %s' % (image_name, host, e)
```

Comment:
```suggestion
            msg = 'Failed to pull %s on %s: Cannot decode JSON' % (image_name, host)
```

`e` is already printed by `exception()`